OpenSim/Simulation/Model/CoordinateLimitForce.h

```diff
@@ -235,8 +235,8 @@ OpenSim_DECLARE_CONCRETE_OBJECT(CoordinateLimitForce, Force);
     // damping to constant values beyond the limits. These are heap allocated
     // and owned here, but we don't want these functions to be copied if a
     // CoordinateLimitForce is copied.
-    SimTK::NullOnCopyUniquePtr<SimTK::Function::Step> _upStep;
-    SimTK::NullOnCopyUniquePtr<SimTK::Function::Step> _loStep;
+    SimTK::ResetOnCopy<std::unique_ptr<SimTK::Function::Step> > _upStep;
+    SimTK::ResetOnCopy<std::unique_ptr<SimTK::Function::Step> > _loStep;
```

Comment:
Just for a little longer. @aymanhab and I are discussing the move; we've both tried newer versions of swig and I'm currently grappling with a backwards incompatibility. However, I want this PR merged before we upgrade swig.